tests/end_to_end/end_to_end_tests.sh

```diff
@@ -5,16 +5,19 @@ set -e
 TESTING_DIRECTORY="$( cd "$(dirname "$0")"/ ; pwd -P )"
 export FEATUREFORM_TEST_PATH=$TESTING_DIRECTORY
 
-echo -e "Exporting FEATUREFORM_HOST='$1' and FEATUREFORM_CERT='$2'\n"
-export FEATUREFORM_HOST=$1
-export FEATUREFORM_CERT=$2
+if [ $# -eq 2 ]; then
```

Comment:
yeah that's confusing. basically, `$#` returns the number of arguments passed into the script. I have this check because we should be able to run the script locally as well as through GitHub actions.

Reply:
Gotcha